api/src/main/resources/workbench.yaml

```diff
@@ -576,6 +576,34 @@ paths:
           required: true
           description: ID of the workspace containing the cohort definition
 
+  /api/v1/workspaces/{workspaceNamespace}/{workspaceId}/materializeCohort:
+    post:
+      tags:
+        - cohorts
+      description: Materializes a cohort for a given CDR version to specified output
+      operationId: "materializeCohort"
+      parameters:
+       - in: path
+         name: workspaceNamespace
+         type: string
+         required: true
+         description: namespace of the workspace that the cohort will be materialized in the context of
+       - in: path
+         name: workspaceId
```

Comment:
So why do you need to workspaceNamespace and workspaceId? Is this to help determine the cdr version?

Reply:
1) It gives a scope for the cohort name.

2) It gives the cdr version for the workspace.